From 605484fd9fc873b9b8d571f398c3cab05c68d860 Mon Sep 17 00:00:00 2001
From: Luca Beltrame <lbeltrame@kde.org>
Date: Thu, 10 May 2018 09:28:15 +0200
Subject: Fix build with Qt 5.11

Some includes added transitively previously now need to be explicitly
put.
---
 src/svnfrontend/blamedisplay.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/svnfrontend/blamedisplay.cpp b/src/svnfrontend/blamedisplay.cpp
index 2d87502..5673181 100644
--- a/src/svnfrontend/blamedisplay.cpp
+++ b/src/svnfrontend/blamedisplay.cpp
@@ -30,6 +30,7 @@
 #include <KTextEdit>
 #include <KTreeWidgetSearchLine>
 
+#include <QAction>
 #include <QBrush>
 #include <QFontDatabase>
 #include <QInputDialog>
-- 
cgit v0.11.2
